### PR TITLE
RELATED: RAIL-2791 prepare CI scripts for gitflow

### PR DESCRIPTION
**Pull request by no23reason**
Split the creation of the next preminor alpha into a separate script.
This is needed, because on gitflow, the next alpha is created before
the minor/major is actually released (it is done on release branch creation).

JIRA: RAIL-2791

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [x] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1666/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3241/ : SUCCESS in 22m 58s

File: common/scripts/ci/run_preminor.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+bump_rc=$?
+
+if [ $bump_rc -ne 0 ]; then
+    echo "Version bump failed. Stopping."
+
+    exit 1
+fi
+
+export TAG_NPM="prerelease"
+${DIR}/do_publish.sh
```
Comment (lupko) [inline common/scripts/ci/run_preminor.sh:24]: is it desired to publish new preminor immediately after bump?

perhaps here we should have the:

```
  NEXT_MINOR=$(get_current_version)	
  # stage all modified json files	
  git ls-files | grep '\.json' | xargs git add	
  git commit -m "Prepare for next minor release ${NEXT_MINOR}"
```

**Review by lupko — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/530/ : SUCCESS in 29s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/790/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3263/ : SUCCESS in 14m 07s

**Pull request merged**